pynamodb/connection/base.py

```diff
@@ -1020,7 +1027,19 @@ def rate_limited_scan(self,
                             if not limit:
                                 return
 
-                    latest_scan_consumed_capacity = data.get(CONSUMED_CAPACITY).get(CAPACITY_UNITS)
+                    if CONSUMED_CAPACITY in data:
+                        latest_scan_consumed_capacity = data.get(CONSUMED_CAPACITY).get(CAPACITY_UNITS)
+                    else:
+                        if allow_rate_limited_scan_without_consumed_capacity:
+                            latest_scan_consumed_capacity = 0
+                        else:
+                            raise ScanError('Rate limited scan not possible because the server did not send back'
```

Comment:
I think the last two sentences are probably overkill here. Anyone interested enough in this will read the settings and find all of the information they need.